Army/Rep/ArmyLoremasterRep.cs

```diff
@@ -0,0 +1,109 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreAdvanced.cs
+//cs_include Scripts/Army/CoreArmyLite.cs
+using Skua.Core.Interfaces;
+using Skua.Core.Models.Items;
+using Skua.Core.Options;
+
+public class ArmyArcangroveRep
+{
+    private IScriptInterface Bot => IScriptInterface.Instance;
+    private CoreBots Core => CoreBots.Instance;
+    private CoreFarms Farm = new();
+    private CoreAdvanced Adv => new();
+    private CoreArmyLite Army = new();
+
+    private static CoreBots sCore = new();
+    private static CoreArmyLite sArmy = new();
+
+    public string OptionsStorage = "ArmyArcangroveRep";
+    public bool DontPreconfigure = true;
+    public List<IOption> Options = new List<IOption>()
+    {
+        sArmy.player1,
+        sArmy.player2,
+        sArmy.player3,
+        sArmy.player4,
+        sArmy.player5,
+        sArmy.player6,
+        sArmy.packetDelay,
+        sCore.SkipOptions
+    };
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+        bot.Options.RestPackets = false;
+
+        Setup();
+
+        Core.SetOptions(false);
+    }
+
+    public void Setup()
+    {
+        if (Farm.FactionRank("Loremaster") >= 10)
+            return;
+
+        Core.PrivateRooms = true;
+        Core.PrivateRoomNumber = Army.getRoomNr();
+        Core.EquipClass(ClassType.Farm);
+        Farm.ToggleBoost(BoostType.Reputation);
+        while (!Bot.ShouldExit && Farm.FactionRank("Loremaster") < 10)
+        {
+            if (!Core.IsMember ? Farm.FactionRank("Loremaster") < 10 : Farm.FactionRank("Loremaster") < 10)
```

Comment:
why u are using conditional operator if the true and false return the same thing?

Reply:
Better approach with the latest commit